src/components/pages/NotificationCenter/NotificationItem.tsx

```diff
@@ -81,13 +81,15 @@ const NotificationContent = ({
   const userId = item.contentParsed?.userId
   const appId = item.contentParsed?.appId
   const you = UserService.getName()
-  const appName = item.contentParsed?.AppName
+  const appName = item.contentParsed?.AppName ?? item.contentParsed?.appName
```

Comment:
@jjeroch If possible can we change this from `appName` to `AppName` which is already present?